packages/core/src/test/awsService/appBuilder/explorer/walkthrough.test.ts

```diff
@@ -9,22 +9,22 @@ import assert from 'assert'
 import {
     openApplicationComposerAfterReload,
     templateToOpenAppComposer,
-} from '../../../awsService/appBuilder/activation'
-import globals from '../../../shared/extensionGlobals'
+} from '../../../../awsService/appBuilder/activation'
+import globals from '../../../../shared/extensionGlobals'
 import {
     RuntimeLocationWizard,
     genWalkthroughProject,
     openProjectInWorkspace,
-} from '../../../awsService/appBuilder/walkthrough'
-import { createWizardTester } from '../../shared/wizards/wizardTestUtils'
-import { fs } from '../../../shared'
-import { getTestWindow } from '../../shared/vscode/window'
-
-import { AwsClis, installCli } from '../../../shared/utilities/cliUtils'
-import { ChildProcess } from '../../../shared/utilities/processUtils'
-import { assertTelemetryCurried } from '../../testUtil'
-import { HttpResourceFetcher } from '../../../shared/resourcefetcher/httpResourceFetcher'
-import { SamCliInfoInvocation } from '../../../shared/sam/cli/samCliInfo'
+} from '../../../../awsService/appBuilder/walkthrough'
+import { createWizardTester } from '../../../shared/wizards/wizardTestUtils'
+import { fs } from '../../../../shared'
+import { getTestWindow } from '../../../shared/vscode/window'
+
+import { AwsClis, installCli } from '../../../../shared/utilities/cliUtils'
+import { ChildProcess } from '../../../../shared/utilities/processUtils'
+import { assertTelemetryCurried } from '../../../testUtil'
+import { HttpResourceFetcher } from '../../../../shared/resourcefetcher/httpResourceFetcher'
+import { SamCliInfoInvocation } from '../../../../shared/sam/cli/samCliInfo'
```

Comment:
The move here is to have the test reflecting the structure of the code. 
There is a backlog AI for this. Since I am adding a new test file, I create a subfolder structure to reflect the source code.

Reply:
I still don't understand the point here. walkthrough is in `appBuilder/walkthrough.ts` but test is in `appBuilder/explorer/walkthrough.test.ts`